game/scripts/vscripts/heroes/antimage/antimage_mana_break.lua

```diff
@@ -0,0 +1,101 @@
+-- Author: Shush
+-- Date: 25/11/2017
+
+-------------------------------
+--   Anti Mage's Mana Break  --
+-------------------------------
+lua_antimage_mana_break = class({})
+LinkLuaModifier("modifier_lua_antimage_mana_break", "antimage/antimage_mana_break", LUA_MODIFIER_MOTION_NONE)
+
+function lua_antimage_mana_break:GetIntrinsicModifierName()
+    return "modifier_lua_antimage_mana_break"
+end
+
+function lua_antimage_mana_break:GetAbilityTextureName()
+   return "antimage_mana_break"
+end
+
+-- Mana break's.. well, mana burn modifier
+modifier_lua_antimage_mana_break = class({})
+
+function modifier_lua_antimage_mana_break:IsHidden() return true end
+function modifier_lua_antimage_mana_break:IsPurgable() return false end
+function modifier_lua_antimage_mana_break:IsDebuff() return false end
+function modifier_lua_antimage_mana_break:RemoveOnDeath() return false end
+
+function modifier_lua_antimage_mana_break:OnCreated()
+    -- Ability properties
+    self.caster = self:GetCaster()
+    self.ability = self:GetAbility()
+    self.hit_sound = "Hero_Antimage.ManaBreak"
+    self.particle_manaburn = "particles/generic_gameplay/generic_manaburn.vpcf"
+
+    -- Ability specials
+    self.damage_per_burn = self.ability:GetSpecialValueFor("damage_per_burn")
+    self.mana_per_hit = self.ability:GetSpecialValueFor("mana_per_hit")
+end
+
+function modifier_lua_antimage_mana_break:OnRefresh()
+    self:OnCreated()
```

Comment:
eeeeeeeeh, better to just update the special values in here than rerun the oncreated.

Reply:
Agreed.